lua/cl_specdm_hud.lua

```diff
@@ -29,106 +29,145 @@ local ammo_colors = {
 local function DrawBg(x, y, width, height, client)
 	local th = 30
 	local tw = 170
+    
 	y = y - th
 	height = height + th
+    
 	draw.RoundedBox(8, x, y, width, height, bg_colors.background_main)
+    
 	local col = bg_colors.innocent
+    
 	if client:IsGhost() then
-	elseif GAMEMODE.round_state != ROUND_ACTIVE then
+	elseif GAMEMODE.round_state ~= ROUND_ACTIVE then
 		col = bg_colors.noround
-	elseif client:GetTraitor() then
-		col = bg_colors.traitor
-	elseif client:GetDetective() then
-		col = bg_colors.detective
-	end
+	else
+        if not ROLES then
+            if client:GetTraitor() then
+                col = bg_colors.traitor
+            elseif client:GetDetective() then
+                col = bg_colors.detective
+            end
+        else
+            col = client.GetRoleData().color
+        end
+    end
+    
 	draw.RoundedBox(8, x, y, tw, th, col)
 end
 
 local Tex_Corner8 = surface.GetTextureID("gui/corner8")
-local function RoundedMeter( bs, x, y, w, h, color)
+
+local function RoundedMeter(bs, x, y, w, h, color)
 	surface.SetDrawColor(clr(color))
-	surface.DrawRect( x+bs, y, w-bs*2, h )
-	surface.DrawRect( x, y+bs, bs, h-bs*2 )
-	surface.SetTexture( Tex_Corner8 )
-	surface.DrawTexturedRectRotated( x + bs/2 , y + bs/2, bs, bs, 0 )
-	surface.DrawTexturedRectRotated( x + bs/2 , y + h -bs/2, bs, bs, 90 )
+	surface.DrawRect(x + bs, y, w - bs * 2, h)
+	surface.DrawRect(x, y + bs, bs, h - bs * 2)
+    
+	surface.SetTexture(Tex_Corner8)
+	surface.DrawTexturedRectRotated(x + bs / 2 , y + bs / 2, bs, bs, 0)
+	surface.DrawTexturedRectRotated(x + bs / 2 , y + h - bs / 2, bs, bs, 90)
+    
 	if w > 14 then
-		surface.DrawRect( x+w-bs, y+bs, bs, h-bs*2 )
-		surface.DrawTexturedRectRotated( x + w - bs/2 , y + bs/2, bs, bs, 270 )
-		surface.DrawTexturedRectRotated( x + w - bs/2 , y + h - bs/2, bs, bs, 180 )
+		surface.DrawRect(x + w - bs, y + bs, bs, h - bs * 2)
+		surface.DrawTexturedRectRotated(x + w - bs / 2 , y + bs / 2, bs, bs, 270)
+		surface.DrawTexturedRectRotated(x + w - bs / 2 , y + h - bs / 2, bs, bs, 180)
 	else
-		surface.DrawRect( x + math.max(w-bs, bs), y, bs/2, h )
+		surface.DrawRect(x + math.max(w-bs, bs), y, bs/2, h)
 	end
 end
 
 local function GetAmmo(ply)
 	local weap = ply:GetActiveWeapon()
-	if not weap or not ply:Alive() then return -1 end
+    
+	if not weap or not ply:Alive() then 
+        return -1 
+    end
```

Comment:
What is this indentation? Shouldn't it be more to the right?
Many of your changed [`if`](https://github.com/Tommy228/TTT_Spectator_Deathmatch/pull/73/files#diff-5b6478f543c044b73e49d5235bf3e5c5R128) and [`timer`](https://github.com/Tommy228/TTT_Spectator_Deathmatch/pull/73/files#diff-dfff33395cd2f31d3eca5431f44bd6efR96) blocks have strange indentation.

If it's not intended I can change it, just wanted to ask before.

Reply:
hmmm, that's strange... I use auto intent, but it looks like that there are some spaces... Definitely not wanted

Reply:
So feel free to change it :)